recording/docker-compose/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+# Common dependencies
+RUN apt-get -y install software-properties-common
+
+# spreed-recording dependencies
+RUN apt-get -y install ffmpeg pulseaudio python3-pip xvfb
+RUN pip3 install --upgrade requests
```

Comment:
This should not be needed, as the package will be updated if needed when running `python3 -m pip install /tmp/recording/`

Reply:
The command `python3 -m pip install /tmp/recording/` updates the locally installed package, and `pip3 install --upgrade requests` updates the globally installed package. Something akin to `/usr/bin` and `/usr/local/bin`.

Even though it works, on a default Ubuntu 20.04 Docker image there are warnings about version mismatches between the locally and globally installed requests package. Updating the globally installed `requests` package takes care of the warnings.

Reply:
Specifically, this is what I get:

```
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.15) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
No configured backends
 * Serving Flask app 'nextcloud.talk.recording.Server'
 * Debug mode: off
INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8082
 * Running on http://172.25.0.2:8082
INFO:werkzeug:Press CTRL+C to quit
```

Reply:
Ah, ok, it seems that I did not get the error because the branch was rooted on 9aed218e, so the pull request that added `requests-toolbelt` as dependency had not been merged yet. Now I see them as well :-)

I would have expected `python3 -m pip install /tmp/recording/` to anyway update the dependencies as needed in `/usr/local` and use them overriding the ones from `/usr/`, but if it does not work that way and an explicit upgrade is needed... so be it :+1: